src/replicache.ts

```diff
@@ -153,6 +144,7 @@ export default class Replicache implements ReadTransaction {
   private _syncInterval: number | null;
   // NodeJS has a non standard setTimeout function :'(
   protected _timerId: ReturnType<typeof setTimeout> | 0 = 0;
```

Comment:
Maybe change this to `pullTimerId` for clarity.

Reply:
This is the sync timer...  I'll rename to make it clearer. We need to revisit this later.